### PR TITLE
TypeOfReact install

### PR DESCRIPTION
**Pull request by Kh-BatErdene**
npm install --dev @types/react === command aar type suulgaw

### CODE REVIEW

File: node_modules/.package-lock.json
Comment (Kh-BatErdene) [inline node_modules/.package-lock.json:1]: node modul s orj irehgui bh ystoi zasaarai

**Review by Kh-BatErdene — COMMENTED** [review]
code bichihtei anhaaraltai bicheerei

**Review by Kh-BatErdene — COMMENTED** [review]
ok

**Review by Moully — APPROVED** [review]
bolson bainaa

**Pull request merged**